src/crafting.rs

```diff
@@ -124,5 +145,10 @@ pub fn print_recipes() {
         item.0.bold().dimmed(),
       )
     }
+    for upgrade in &recipe.upgrades_needed {
+      if upgrade as &str == "fire" {
```

Comment:
Why do you need to cast the `upgrade` variable to `&str`? 😮

Reply:
ah, `upgrade` is `str` and it was complaining `can't compare &str with str` 🤷‍♀